completions/bash/crio

```diff
@@ -39,6 +39,7 @@ h
 --ctr-stop-timeout
 --decryption-keys-path
 --default-capabilities
+--default-container-min-memory
```

Comment:
@haircommander, it's helpful to have it, and it also follows the pattern of other such options (e.g., `--runtimes`).

We still respect the built-in limit of 12 MiB if no override is provided.

Reply:
The separate cli command may be redundant because the per-runtime option is available. If the default option is not provided, it will be overridden(maybe unexpected by the user 🤔 ); on the other hand, default options reduce unexpected behavior and provide a more declarative way.

Reply:
I personally think we should be pretty strict about the top-level options we provide in the configuration. it's already quite large. I am a semi strong NACK on it, though I would defer to other maintainers if you really want it.

> it's helpful to have it, and it also follows the pattern of other such options (e.g., --runtimes).

I would say it's closer to the `runtime_type` field. We implicitly assume the runtime_type is `oci` unless otherwise specified. We don't have a top-level config field `default_runtime_type` because it's fine to assume `oci` unless otherwise specified

Reply:
@haircommander, sounds good!

So, @roman-kiselenko, we need to drop the command-line switches, and only keep the configuration changes, such that we have the internal default that can be overridden on the runtime and per-runtime levels. So, things will be even simpler.

Reply:
I'm fine with keeping the cmdline switches for `--runtimes` fwiw, just not having the top level config field generally